modules/apps/headless/headless-delivery/headless-delivery-api/src/main/java/com/liferay/headless/delivery/dto/v1_0/util/ContentFieldUtil.java

```diff
@@ -91,10 +91,14 @@ public static ContentField toContentField(
 							return null;
 						}
 
-						Map<String, ContentFieldValue> map = new HashMap<>();
-
 						Value value = ddmFormFieldValue.getValue();
 
+						if (value == null) {
+							return new HashMap<>();
+						}
```

Comment:
suggestion: 

On line 77 and 91 it's returning a null. maybe returning a null is more standardised on this.

then you can put on the 99 line if unified

Reply:
<img src="https://media0.giphy.com/media/gk9N8cqyCdNgCWG7MC/giphy.gif"/>

Reply:
❤️

Reply:
Changes made in the past :P